File: Modul-4/Soal 4/PRAK404-2410817320015-NazwaAmanda.c
#include <stdio.h>

int main(){
    int n;
    float NilaiPertama, NilaiKedua;
    while (1){
        printf("\nPilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n");
        printf("Masukkan pilihan : ");
        scanf("%d", &n);

        if (n < 1 || n > 5) {printf("Input anda salah, silahkan coba lagi"); continue;
    } else if (n == 5) {printf("Terimakasih, telah menggunakan kalkulator Nazwa Amanda"); break; }

    printf("Masukkan nilai pertama : ");
    scanf("%f", &NilaiPertama);
    printf("Masukkan nilai kedua : ");
    scanf("%f", &NilaiKedua);

    if (n == 1) { printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f", NilaiPertama, NilaiKedua, NilaiPertama + NilaiKedua);
    } else if (n == 2) { printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f", NilaiPertama, NilaiKedua, NilaiPertama - NilaiKedua);
    } else if (n == 3) { printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f", NilaiPertama, NilaiKedua, NilaiPertama * NilaiKedua);
    } else if (n == 4) { printf("Hasil pembagian anatara %.2f dengan %.2f adlah %.2f", NilaiPertama, NilaiKedua, NilaiPertama / NilaiKedua);
    }
}
return 0;
}

File: Modul-4/Soal 4/PRAK404-2410817320015-NazwaAmanda.py
while True:
    print("\nPilih program\n1. Penjumlahan\n2. Penfurangan\n3. Pembagian\n5. Exit")
    n = int(input("Masukkan Pilihan : "))

    if n < 1 or n > 5:
        print("Input anda salah, silahkan coba lagi")
        continue
    elif n == 5:
        print("Terimakasih, telah menggunakan kalkulator Nazwa Amanda")
        break

    NilaiPertama = float(input("Masukkan nilai pertama : "))
    NilaiKedua = float(input("Masukkan nilai kedua : "))

    if n == 1:
     Hasil = NilaiPertama + NilaiKedua
     print(f"Hasil penjumlahan antara {NilaiPertama:.2f} dengan {NilaiKedua:.2f} adalah {Hasil:.2f}")
    elif n == 2:
       Hasil = NilaiPertama - NilaiKedua
       print(f"Hasil pengurangan antara {NilaiPertama:.2f} dengan {NilaiKedua:.2f} adalah {Hasil:.2f}")
    elif n == 3:
       Hasil = NilaiPertama * NilaiKedua
       print(f"Hasil perkalian antara {NilaiPertama:.2f} dengan {NilaiKedua:.2f} adalah {Hasil:.2f}")
    elif n == 4:
       Hasil = NilaiPertama / NilaiKedua
       print(f"Hasil pembagian anatara {NilaiPertama:.2f} dengan {NilaiKedua:.2f} adalah {Hasil:.2f}")